src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusHttpListener.cs

```diff
@@ -35,16 +36,18 @@ internal sealed class PrometheusHttpListener : IDisposable
         /// <summary>
         /// Initializes a new instance of the <see cref="PrometheusHttpListener"/> class.
         /// </summary>
-        /// <param name="exporter"><see cref="PrometheusExporter"/>The exporter instance.</param>
-        /// <param name="options"><see cref="PrometheusHttpListenerOptions"/>The configured HttpListener options.</param>
-        public PrometheusHttpListener(PrometheusExporter exporter, PrometheusHttpListenerOptions options)
+        /// <param name="exporter"><see cref="PrometheusExporter"/>The path to use for the scraping endpoint.</param>
+        /// <param name="scrapeEndpointPath">The configured HttpListener options.</param>
+        /// <param name="uriPrefixes">The URI (Uniform Resource Identifier) prefixes to use for the http listener.</param>
+        public PrometheusHttpListener(PrometheusExporter exporter, string scrapeEndpointPath, IEnumerable<string> uriPrefixes)
```

Comment:
I don't totally agree with this change philosophically 🤣 Passing in the options makes it easier to add new things in the future. If you want to validate that the `scrapeEndpointPath` is valid why not just modify [the setter](https://github.com/open-telemetry/opentelemetry-dotnet/blob/ed0450ef0ef5178cb29cb8908c3ff120a053caad/src/OpenTelemetry.Exporter.Prometheus.HttpListener/PrometheusHttpListenerOptions.cs#L33) kind of like what `UriPrefixes` is doing?

Reply:
I reverted changes related to options in this file.
I agree that it is better to have options here, but the new option is not used internally by `PrometheusHttpListener`. It is needed by another class, so I would like to make it clear by the contract without introducing any new concepts/additional breaking changes in the API.
Please let me know if you have any idea how to improve it, or we can keep it as it is for now.

Reply:
I'm OK with this. IIRC `PrometheusExporter` is all `internal` now so the fact that there are two classes is kind of an implementation detail. I think we could clean it up a bit and achieve some code re-use between the two projects but we can do as a follow-up.

Reply:
@Kielek I opened a PR in your fork into the branch behind this PR: https://github.com/Kielek/opentelemetry-dotnet/pull/1

I renamed `PrometheusExporterOptions` -> `PrometheusAspNetCoreOptions` and introduced `internal PrometheusExporterOptions` which is shared by both projects. This achieves better code reuse and also allows us to in the future expose `PrometheusExporter` + friends if we ever need to. Essentially I think the AspNetCore project owning `PrometheusExporterOptions` would be a problem down the road.

Anyway if you merge that PR it should become part of this one automatically and not cause you any conflicts 🤞

Reply:
Thank you! PR merged. Some minor tweaks added :)